presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -699,7 +701,7 @@ public void testCorrelatedSubqueries()
         assertPlan(
                 "SELECT orderkey FROM orders WHERE 3 = (SELECT orderkey)",
                 OPTIMIZED,
-                any(
+                anyTree(
```

Comment:
any(source pattern) matches one node with source pattern.
anyTree(source pattern) matches one or more nodes with source matches source pattern.

Before optimization, the query plan is output <- filter <- scan
After optimization, the query plan is output <- project (orderkey := 3) <- filter <- scan

Reply:
would be nice to add project instead of anyTree there since we have a specific expectation. i guess the challenge for all the tests (but particularly the ones specific to the new optimizer) is that its hard to tell which rows are actually output from the scan.  from the tests its hard to tell that we're dropping columns that we don't

Reply:
Fixed